src/tools/status_tool.cpp

```diff
@@ -698,6 +700,40 @@ int StatusTool::PrintChunkserverStatus(bool checkLeftSize) {
     std::map<PoolIdType, std::vector<uint64_t>> poolChunkLeftSize;
     std::map<PoolIdType, std::vector<uint64_t>> poolWalSegmentLeftSize;
     std::vector<ChunkServerIdType> offlineCs;
+    bool useWalPool = false;
+    bool useChunkFilePoolAsWalPool = true;
+
+    if (!poolChunkservers.empty()) {
+        ChunkServerInfo chunkserver = poolChunkservers.begin()->second[0];
+        std::string csAddr = chunkserver.hostip()
+                            + ":" + std::to_string(chunkserver.port());
+        // check whether use chunkfilepool
+        std::string metricValue;
+        std::string metricName = GetUseWalPoolName(csAddr);
+        MetricRet res = metricClient_->GetConfValueFromMetric(csAddr,
+                                            metricName, &metricValue);
+        if (res != MetricRet::kOK) {
+            std::cout << "Get use chunkfilepool conf "
+                    << csAddr << " fail!" << std::endl;
+            ret = -1;
```

Comment:
Don't need other Exception handling？

Reply:
> Don't need other Exception handling？

if get the conf metric failed, maybe the configuration file is uncorrect